codes/models/run_projection_with_multiple_gammas.R

```diff
@@ -0,0 +1,98 @@
+library(tidyverse)
+library(patchwork)
+list.files("codes/models", full.names = TRUE) %>% 
+  str_subset("data_and_model_projection") %>% 
+  walk(source)
+
+gamma_loop_df <- crossing(
+  tibble(country =   c("china", "iran", "korea", "italy", "france", "usa"),
+       urcountry = c("China", "Iran", "South Korea", "Italy", "France", "United States of America"),
+       full_data = list(china_data, iran_data, korea_data, italy_data, france_data, usa_data),
```

Comment:
I get an error that none of this data can be found. Do we need to have a line that sources the `generate_data` scripts above this?